### ISSUE TITLE
How to start wharfee on Windows with Docker Toolbox? <eom>

### ISSUE THREAD

**Issue by Tset-Noitamotua** [j-bennet/wharfee#126]
So to answer my own question:
 
I need to type `wharfee` into Docker Quickstart Terminal.

**Comment by j-bennet** [comment]
@Tset-Noitamotua, I don't have a windows machine to test with, but in theory it's pretty much the same as mac, because mac also used a VM, either docker-machine or boot2docker. Is there a particular issue you're facing?

**Comment by Tset-Noitamotua** [comment]
No, no particular issue. I just wondered how to start it after installing because I found no mention about it in your readme.  Also I did not notice on your screenshots that one have to type `wharfee` into terminal to start it because 'wharfee' is already in your prompt so I asumed that wharfee is already up and running in that screenshot. I just overlooked the input.

![image](https://cloud.githubusercontent.com/assets/5635462/11295454/c839f0b2-8f6b-11e5-98bc-a3db59a04b90.png)



So to answer my own question *"How to start wharfee on Windows with Docker Toolbox?"*: 
I need to type `wharfee` into Docker Quickstart Terminal.



This it how it looks on Windows in Docker Quickstart Terminal:

![image](https://cloud.githubusercontent.com/assets/5635462/11295282/b7f5a832-8f6a-11e5-98cf-cb77c41e34b1.png)
